teuthology-docs/config/definitions/teuthology-docs.yml

```diff
@@ -34,3 +34,12 @@
     builders:
       - shell:
           !include-raw ../../build/build
+
+    # publish docs to docs.ceph.com/teuthology/docs
+    publishers:
+      - ssh:
+          site: 'docs.ceph.com'
+          target: 'teuthology/docs/'
+          source: '.tox/docs/tmp/html/**'
+          remove-prefix: '.tox/docs/tmp/html/'
+          clean-remote: true
```

Comment:
I would rather not do this, because if the transfer gets cut in the middle for whatever reason it will leave us with a broken docs site